vapi/flatpak.vapi

```diff
@@ -295,10 +295,10 @@ namespace Flatpak {
 		MMC,
 		NETWORK
 	}
-	[CCode (cheader_filename = "flatpak.h", cprefix = "FLATPAK_TRANSACTION_ERROR_DETAILS_NON_", type_id = "flatpak_transaction_error_details_get_type ()")]
+	[CCode (cheader_filename = "flatpak.h", cprefix = "FLATPAK_TRANSACTION_ERROR_DETAILS_", type_id = "flatpak_transaction_error_details_get_type ()")]
 	[Flags]
 	public enum TransactionErrorDetails {
-		FATAL
+		NON_FATAL
```

Comment:
Does it make sense to fix this file here, or is it generated/imported from somewhere else?

Reply:
This is a generated file and I've been fixing issues upstream in flatpak that cause this file to not be generated correctly. However this one is just an issue with the generator, so I think this one is fine to manually fix like this.